src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -134,7 +134,7 @@ public TimeSpan NonceLifetime
             {
                 if (value <= TimeSpan.Zero)
                 {
-                    throw LogHelper.LogException<ArgumentOutOfRangeException>(LogMessages.IDX10105, value);
+                    throw LogHelper.LogArgumentException<ArgumentOutOfRangeException>("value", LogMessages.IDX10105, value);
```

Comment:
In this case, because it's a property, you'll get "Parameter name: value" along with the specialized error message. This is generally consistent with the parameter name for .NET properties.